homeassistant/components/aranet/sensor.py

```diff
@@ -74,6 +74,11 @@ class AranetSensorEntityDescription(SensorEntityDescription):
         native_unit_of_measurement=UnitOfPressure.HPA,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    "status": AranetSensorEntityDescription(
+        key="concentration_status",
+        name="Concentration Status",
```

Comment:
I'm still pondering the best name for this sensor. Some options I've considered:

- Concentration Status
- Concentration Level
- Threshold Level
- Threshold Status

From the [Aranet Radon Plus FAQs](https://aranet.com/en/home/products/aranet-radon-sensor):

> #### What is the purpose of the color-coded indication bars on the sensor?
> The color-coded bars (green, yellow, red) indicate the radon concentration levels and warn you of unhealthy levels. The thresholds for these bars can be adjusted via the Aranet Home app. The default thresholds are set following World Health Organization (WHO) Radon guidelines.